libp2p/protocols/pubsub/peertable.nim

```diff
@@ -53,3 +53,10 @@ func peers*(table: PeerTable, topic: string): int =
     except KeyError: raiseAssert "checked with in"
   else:
     0
+
+func outboundPeers*(table: PeerTable, topic: string): int =
+  if topic in table:
+    try: table[topic].countIt(it.outbound)
```

Comment:
out of curiosity, is the `try` needed? `if topic in table` should protect from trying to access a non existing key? so `KeyError` should never happen?

Reply:
It should never happen indeed, but the compiler thinks it can

I think this all proc could just be `table.getOrDefault(topic).countIt(it.outbound)`, but wanted to keep it consistent with the one above